### PR TITLE
Stage/Phase Rocket Chip Bump

### PR DESCRIPTION
**Pull request by davidbiancolin**
See ucb-bar/chipyard#503

### CODE REVIEW

File: src/main/scala/ArianeTile.scala [hunk 1]
```diff
@@ -268,16 +268,14 @@ class ArianeTileModuleImp(outer: ArianeTile) extends BaseTileModuleImp(outer){
     //outer.traceSourceNode.bundle <> core.io.trace_o.asTypeOf(outer.traceSourceNode.bundle)
 
     for (w <- 0 until outer.arianeParams.core.retireWidth) {
-      outer.traceSourceNode.bundle(w).clock     := core.io.trace_o(traceInstSz*w + 0).asClock
```
Comment (abejgonzalez) [inline src/main/scala/ArianeTile.scala:271]: All you need to do to make this work is to uncomment the `reset` and `valid` lines (no need to shift the indexes). I would look at the #2 for the correct solution. Also, I think we can just delete the "fake" Rocket params like he does.

**Review by abejgonzalez — CHANGES_REQUESTED** [review]

File: src/main/scala/ArianeTile.scala (hunk 1 above)
Comment (davidbiancolin) [reply src/main/scala/ArianeTile.scala:271]: Ah derp. Ok.

Comment (davidbiancolin) [reply src/main/scala/ArianeTile.scala:271]: I wasn't thinking here.

**Review by abejgonzalez — APPROVED** [review]
LGTM

**Comment by abejgonzalez** [comment]
Merge at your leisure.

**Pull request merged**